monkey/monkey_island/cc/environment/test_environment_config.py

```diff
@@ -123,3 +123,14 @@ def test_generate_default_file(config_file):
 def test_data_dir():
     environment_config = EnvironmentConfig(WITH_DATA_DIR)
     assert environment_config.data_dir == "/test/data/dir"
+
+
+def set_home_env(monkeypatch, tmpdir):
+    monkeypatch.setenv("HOME", str(tmpdir))
+
+
+def test_data_dir_abs_path_from_file(monkeypatch, tmpdir):
+    set_home_env(monkeypatch, tmpdir)
```

Comment:
Why not make `set_home_env` a fixture?

Reply:
In this case, both `set_home_env()` and the test need to use the same `tmpdir`. Passing a tmpdir to a function that sets an environment variable, to me, seems easier to understand than a fixture that relies on another fixture and returns a directory. Since it's only used in one test that's 3 lines long, it probably doesn't matter much one  way or the other, but it's something to revisit if this test suite is expanded.